backend/infrahub/core/schema/schema_branch.py

```diff
@@ -1425,7 +1425,8 @@ def process_inheritance(self) -> None:
                 node.validate_inheritance(interface=generic_kind_schema)
 
                 # Store the list of node referencing a specific generics
-                generics_used_by[generic_kind].append(node.kind)
+                if node.namespace != "Internal":
+                    generics_used_by[generic_kind].append(node.kind)
```

Comment:
This avoid having `Internal*` in generic's `used_by`. It does not seem to break anything and avoid exposing nodes in list while they are actually not exposed in the schema served by the REST API `/api/schema` endpoint.